### PR TITLE
22.2.1rc1

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ <skjnldsv@protonmail.com>

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -524,35 +524,35 @@
...
+				'signature' => 'gFblqxh+w7sGgWkYa/joydKyexmP7g2lZmljjXdifxPk9vf167UGAK2nQgHgge15
+rqBiBPDne82q7ysPBSBBzzw2qyWYK4Y629zcDVEurOxsxV/yzh2lrvPZiCrrHltG
+0cMqyOeUVBtKGGIEp34LYtG7M1wr0b92Jy1IMzanFP8NAG8/N4pYNevRm1JjB9BY
+NVp19RztO8m/mPuRFTXSNZ1SDC6C3yMWrST07rjHOymZJSa8CJyroK/BfQdVonS/
+EN0BxBBUpY5/ZBOvCIBIy0LcP/JpgTkdCim2ooxbAwAof1yvTuubkTGmHQNzoVnv
+ITFtk0PeXc1HZzFZ65CdQw==',
 			],
 		],
 
 		'21.0.5' => [
```
Comment (skjnldsv) [inline config/config.php:542]: Will change once we have 21.0.6 stable?

Comment (blizzz) [reply config/config.php:542]: beta should feed beta

Comment (skjnldsv) [reply config/config.php:542]: What does that means? Should I change it now?

**Review by blizzz — APPROVED** [review]

**Comment by skjnldsv** [comment]
`Upgrades / Upgrade from 21.0.6 to 22.2.1 RC1 (pull_request) Failing after 10s — Upgrade from 21.0.6 to 22.2.1 RC1` fails because there are no `nextcloud-21.0.6.zip` to download

**Comment by blizzz** [comment]
> `Upgrades / Upgrade from 21.0.6 to 22.2.1 RC1 (pull_request) Failing after 10s — Upgrade from 21.0.6 to 22.2.1 RC1` fails because there are no `nextcloud-21.0.6.zip` to download

why does it not happen for the other rcs?

**Comment by skjnldsv** [comment]
> why does it not happen for the other rcs?

Because we don't have an upgrade bath for latest-rc to above-major-rc :thinking: 
We actually never did before.
We only enabled the upgrade from stable 
https://github.com/nextcloud/updater_server/blob/3a0cb004af0edeed67f447d65e12023457e8ec62/config/config.php#L525
https://github.com/nextcloud/updater_server/blob/3a0cb004af0edeed67f447d65e12023457e8ec62/config/config.php#L559
https://github.com/nextcloud/updater_server/blob/3a0cb004af0edeed67f447d65e12023457e8ec62/config/config.php#L593

**Comment by blizzz** [comment]
😕 I am getting old.

line 525 needs a bump however

**Comment by skjnldsv** [comment]
> line 525 needs a bump however

Well, I don't think so. It's like the others, no?

**Comment by blizzz** [comment]
> > line 525 needs a bump however
> 
> Well, I don't think so. It's like the others, no?

but you need to be first on the latest maintenance release to do the major upgrade

**Comment by AndyScherzinger** [comment]
I guess we can close this one, right? because we need to push for 22.2.2 :(

**Comment by blizzz** [comment]
> I guess we can close this one, right? because we need to push for 22.2.2 :(

yes, this is obsolete

**Pull request closed without merging**